routes/postsRouter.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const knex = require('../db/client');
+const cookieParser = require('cookie-parser');
+
+const router = express.Router()
+
+router.get('/', (req,res) => {
+    knex('posts')
+    .orderBy('created_at', 'desc')
+    .then(posts => {
+      let username=req.cookies.username;
+      res.render('posts/', {posts: posts, username:username})
+    })
+  })
+
+
+  // render new post page
+router.get('/new',(req,res) => {
+  let username=req.cookies.username;
```

Comment:
`const username`. Please make sure you understand the difference between `const` and `let`.